pganonymizer/providers.py

```diff
@@ -1,4 +1,5 @@
 import random
+import operator
```

Comment:
Could you please move this import to line 1 to preserve the alphabetically import order?